atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -420,6 +421,12 @@ def get_build(self, metadata, worker_metadatas):
         if flatpak_source_info is not None:
             extra['image'].update(flatpak_source_info.koji_metadata())
 
+        if koji_task_id:
```

Comment:
Looks like `get_koji_task_owner` should be able to handle falsy `koji_task_id`.
This check is redundant in that case.

Reply:
I added that guard when I added the index by 'name'. You can't index None.

Reply:
@bfontecc007 why put `default=None` then if you know you'll get an exception when you actually get `None` back? Is that desirable if your task id comes up with nothing?

Reply:
@sosiouxme I don't really remember in great detail but maybe I thought it was an easier to interpret exception than the one you'd get if you tried to index string '' by 'name.'

Reply:
@sosiouxme is right, that doesn't make any sense.

It looks to me like it should be `get_koji_task_owner()` that does the ['name'] look-up, not the caller of that function.